evm/its.js

```diff
@@ -419,15 +430,32 @@ async function processCommand(_axelar, chain, chains, action, options) {
                 return;
             }
 
-            const data = [];
+            if (itsVersion === '2.1.1') {
```

Comment:
function is becoming complex and unreadable with all this nesting, either simplify or need a helper